odl/trafos/wavelet.py

```diff
@@ -56,13 +110,13 @@ def coeff_size_list(shape, nscales, wbasis, pad_mode):
     nscales : int
         Number of scales in the multidimensional wavelet
         transform.  This parameter is checked against the maximum number of
-        scales returned by ``pywt.dwt_max_level``. For more information
+        scales returned by `pywt.dwt_max_level`. For more information
         see the `PyWavelets documentation on the maximum level of scales
         <http://www.pybytes.com/pywavelets/ref/\
 dwt-discrete-wavelet-transform.html#maximum-decomposition-level\
 -dwt-max-level>`_.
 
-    wbasis : ``pywt.Wavelet``
+    wbasis : {string, `pywt.Wavelet`}
```

Comment:
Curly brackets are used for a fixed range of *values*, not *types*.
Can we link to `pywt.Wavelet`?